sdk/core/Azure.Core.Experimental/src/Variant/Value.PackedDateTimeOffset.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+
+namespace Azure {
+    public readonly partial struct Value
+    {
+        private readonly struct PackedDateTimeOffset
+        {
+            // HHHHHMMT TTT...
+            //
+            // HHHHH   - hour bits 1-31
+            // MM      - minutes flag
+            // T       - ticks bit
+            //            00 - :00
+            //            01 - :15
+            //            10 - :30
+            //            11 - :45
```

Comment:
We should expand on these comments a bit including the valid date range we can represent.

Reply:
I will improve up the comments and docs in a subsequent PR once the implementation/APIs stabilize.